### DISCUSSION TITLE
Smart Connections API Error:: TypeError: Cannot read properties of undefined (reading '0')

### DISCUSSION THREAD

**Discussion by tcttcttct** [brianpetro/obsidian-smart-connections#323] [Q&A]
As title says. I am getting this error when trying to chat: "Smart Connections API Error:: TypeError: Cannot read properties of undefined (reading '0')". 
The chat just keeps putting ellipses and doesn't have any output. 
I did hook up my OpenAI API Key and tested it. 
Not sure how to troubleshoot this. 
Thanks!

**Comment by egouldo** [comment]
I'm having this problem too. I tested my API key and got a warning that there was a problem with it. However, even when I create a brand new API key both issues persist. The problem seems to have started after updating obsidian to 1.4.5 with the new properties feature.

**Comment by rpatrik96** [comment]
I also have the same issue, even with a new API key.
